Maths/FindHcf.js

```diff
@@ -3,7 +3,12 @@
     More about HCF:
         https://en.wikipedia.org/wiki/Greatest_common_divisor
  */
-
+/**
+ *
+ * @param {Number} x
+ * @param {Number} y
+ * @returns {(string|number)}
```

Comment:
This should be explained; ideallly the errors should be thrown instead of returned.